src/charm.py

```diff
@@ -577,8 +583,8 @@ def _install_deps(self) -> None:
         execute_command(["/usr/bin/snap", "refresh", "lxd", "--channel=latest/stable"])
         execute_command(["/snap/bin/lxd", "waitready"])
         execute_command(["/snap/bin/lxd", "init", "--auto"])
-        execute_command(["/usr/bin/chmod", "a+wr", "/var/snap/lxd/common/lxd/unix.socket"])
         execute_command(["/snap/bin/lxc", "network", "set", "lxdbr0", "ipv6.address", "none"])
+        execute_command(["/usr/bin/chmod", "a+wr", "/var/snap/lxd/common/lxd/unix.socket"])
```

Comment:
I believe it's out of the scope of this PR but why is this needed?

Reply:
I guess this is to call lxc command without sudo, but since the charm runs as root, I don't think this is technically necessary.

Reply:
I just removed that line and it seems to be fine.